utils/forest_utils/src/net/download.rs

```diff
@@ -76,10 +75,10 @@ impl FetchProgress<DownloadStream> {
 
         let response = client.get(url.as_str().try_into()?).await?;
 
-        let pb = ProgressBar::new(total_size);
+        let mut pb = ProgressBar::new(total_size);
         pb.message("Downloading/Importing snapshot ");
-        pb.set_units(crate::io::progress_bar::Units::Bytes);
-        pb.set_max_refresh_rate(Some(Duration::from_millis(500)));
+        pb.set_bytes();
+        pb.set_max_refresh_rate(20);
```

Comment:
Updating every 500 milliseconds is the same as 2 hertz, not 20 hz. Do we need to update 20 times per second rather than just 2 times per second?

Reply:
Typo. Fixed